packages/agent-core/src/types/thought-stream.ts

```
@@ -42,6 +42,7 @@ export interface CheckpointEvent {
 }
 
 /** Options for creating a ThoughtStreamHandler instance */
+// eslint-disable-next-line @typescript-eslint/no-empty-object-type -- intentional extension point for future options
 export interface ThoughtStreamOptions {
   // Currently no options needed, but interface provided for future extensibility
 }
```

Biome (2.3.14)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)